best_practices/forms.rst

```diff
@@ -160,8 +160,8 @@ thing in one line to rendering each part of each field independently. The
 best way depends on how much customization you need.
 
 One of the simplest ways - which is especially useful during development -
-is to render the form tags and use ``form_widget()`` to render all of the
-fields:
+is to render the form tags and use Twig ``form_widget()`` function to render
```

Comment:
I would use either "`form_widget()` Twig function" or just "`form_widget()` function". I like the second one most, as "Twig" doesn't add anything more to the sentence.